patches.json

```diff
@@ -135,7 +135,7 @@
       "2.1.4 - 2.2.5": "MAGECLOUD-2445__do_not_run_cron_when_it_is_disabled__2.1.4.patch"
     },
     "Zendframework1 should use TLS 1.2": {
-      ">=2.1.4 <2.3": "MAGECLOUD-2521__zendframework1_use_TLS_1.2.patch"
+      ">=2.1.4 <2.3": "MAGECLOUD-2521__zendframework1_use_TLS__1.2.patch"
```

Comment:
This change is not required - TLS 1.2 is the version of TLS, not sure why we need double underscore preceding 1.2? I guess you do that for Magento versions where patch applies - here it's not the case.

Reply:
Renamed to MAGECLOUD-2521__zendframework1_use_TLS_1.2__2.1.4.patch 
Thanks